app/scripts/modules/app/src/app.ts

```diff
@@ -17,25 +16,17 @@ import { KAYENTA_MODULE } from '@spinnaker/kayenta';
 import { TITUS_MODULE } from '@spinnaker/titus';
 import { ECS_MODULE } from '@spinnaker/ecs';
 import '@spinnaker/cloudfoundry';
-import { AZURE_MODULE } from '@spinnaker/azure';
-import { HUAWEICLOUD_MODULE } from '@spinnaker/huaweicloud';
-import { TENCENTCLOUD_MODULE } from '@spinnaker/tencentcloud';
```

Comment:
These modules are not published to npm, so removing these requires.

Reply:
I'm not sure why these packages weren't published to begin with. We can probably add them back as `file:` dependencies if folks rely on it.